clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -442,6 +442,16 @@
                         "S999"
                     ],
                     "uniqueItems": true
+                },
+                "cobol-lsp.serverType": {
+                    "type": "string",
+                    "enum": ["JAVA", "NATIVE"],
```

Comment:
@zeibura @zimlu02 @ishche  Updated

Reply:
![image](https://user-images.githubusercontent.com/69206564/187447475-774d0784-38c4-423b-8960-18ccbe29deb2.png)

Reply:
Description updated to - COBOL LS server type